### DISCUSSION TITLE
What does this ICON mean?

### DISCUSSION THREAD

**Discussion by Untouchablez** [Hekili/hekili#1332] [Q&A]
Hi All, 

Cant for the life of me find out what this icon means, is there a list to them or can someone tell me what this is please?

Thanks

![image](https://user-images.githubusercontent.com/37020224/148682113-a1fa9fab-d09b-457b-938e-92a17a915916.png)

**Comment by SpatenLa** [comment]
https://discord.gg/3cCTFxM
place for questions like that
